packages/app/src/app/pages/Dashboard/Content/routes/PathedTemplates/Navigation/elements.ts

```diff
@@ -7,7 +7,11 @@ export const Container = styled.div`
   align-items: center;
 `;
 
-export const NavigationLink = styled(Link)<{ first: boolean; last: boolean }>`
+export const NavigationLink = styled(Link)<{
+  first: boolean;
+  last: boolean;
+  teamId?: string;
```

Comment:
@SaraVieira Why passing `teamId` if you're not using it? 🤔

Reply:
I removed it :)